rest/data/project_test.go

```diff
@@ -65,6 +65,27 @@ func getMockProjectSettings() model.ProjectSettingsEvent {
 	}
 }
 
+func getProjectRef() *model.ProjectRef {
```

Comment:
is there a benefit to factoring these out?

Reply:
not really, I refactored this so a `ProjectRef` is created inside of the test function